### PR TITLE
Add 120 additional icebreakers - Resolve #5003.ts

### LINKED ISSUE #5003: Add new Icebreakers to Parabol
We recently upgraded our resource of 127 icebreaker questions to 237 icebreaker questions!

### Acceptance criteria

Let's include the additional icebreakers in the product so teams around the world can benefit from them!

https://www.parabol.co/resources/icebreaker-questions

### Estimated effort

- 4 hours

### PR DESCRIPTION
**Pull request by garethaledavies**
Adding 120 new icebreaker questions to the product.

### CODE REVIEW

**Comment by nickoferrall** [comment]
I really like the new questions. An interesting icebreaker makes such a difference to the meeting. 

It looks like the Typecheck is failing as we need to use backticks (i.e. ``), instead of regular quotes. When typing the sentence: `‘What's one of your recent pet peeves?’`, the array thinks the question is just `What` as it mistakes the apostrophe as the end of the string. 

If you could add the backticks, I'll merge it into master in time for this week's ship!

**Comment by mattkrick** [comment]
ooo this is gonna need some massaging:
- most of the lines are wrapped in single smart quotes instead of the grave accent
- some of the text body uses plain single quotes & some use smart single quotes

I'd suggest we use single quotes to wrap the lines & use smart single quotes for apostrophes, since the curve looks a little nicer

File: packages/client/utils/makeCheckinGreeting.ts [hunk 1]
```diff
@@ -175,7 +175,127 @@ const questions = [
...
+  'What do your family think you do for work?',
+  'What’s a book that affected you so deeply (emotionally or intellectually) that you still regularly think about it?',
+  'What’s something that doesn’t deserve the hate it gets?',
+  'What’s a simple life hack that everyone can benefit from?',
+  'What is a lesson you feel you learned too late in life?',
+  'What’s your spirit animal this week?',
+  'If buying groceries were a game, what would be one of the loading screen tips?',
+  'What’s one of your recent pet peeves?',
+  'How would your best friend describe you?’,
+  'Congratulations! You’ve been chosen to represent your country in a global competition. What sport or activity are you doing?'
```
Comment (nickoferrall) [inline packages/client/utils/makeCheckinGreeting.ts:298]: Almost there! It looks like Typecheck is complaining about line 298. I think the quote mark at the end of line 297 is causing the mischief

**Comment by nickoferrall** [comment]
Great work! Looking forward to seeing these new icebreakers in action

**Pull request merged**